src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.page {
+  --big-bar-height: 70px;
+  --big-bar-pleft: 62px;
+  --big-bar-border: 1px;
+  --big-search-icon-top: calc(var(--big-bar-height) / 2);
+
+  --small-bar-height: 42px;
+  --small-bar-pleft: 33px;
+  --small-bar-border: 1px;
```

Comment:
this is the same as `--big-bar-border`. no need to repeat it

Reply:
Fixed.